packages/terra-form/docs/NumberField.md

```diff
@@ -0,0 +1,24 @@
+# Number Field
+
+Number Field component for rendering text inputs with standard structured style defaulted to type=number. Also exposes attributes specific to that input type. Uses the input component with the type set to number.
```

Comment:
The way this is worded makes me think the input type could be something other than a number. Possibly reword?